src/components/document/sort-work-view.tsx

```diff
@@ -67,13 +69,24 @@ export const SortWorkView: React.FC = observer(function SortWorkView() {
     return sortedSectionLabels.map(sectionLabel => documentMap.get(sectionLabel));
   };
 
-  function customSort(a: string, b: string) {
-    const parseName = (name: string) => {
-      const [lastName, firstName] = name.split(", ").map((part: string) => part.trim());
-      return { firstName, lastName };
+  function customSort(a: any, b: any) { //sort by last name alphabetically
+    const parseName = (name: any) => {
+      const [lastName, firstName] = name.split(", ").map((part: any) => part.trim());
+      const lastNameNum = parseInt(lastName, 10);
+      return {
+        firstName,
+        lastName,
+        isNumericLastName: !isNaN(lastNameNum),
+        lastNameNum
+      };
     };
     const aParsed = parseName(a);
     const bParsed = parseName(b);
+    if (aParsed.isNumericLastName && bParsed.isNumericLastName) {
+      return aParsed.lastNameNum - bParsed.lastNameNum;
+    }
+    if (aParsed.isNumericLastName) return -1;
+    if (bParsed.isNumericLastName) return 1;
```

Comment:
We shouldn't do anything special for numeric last names. Only demo users have numeric last names, if a real user has a numeric last name we shouldn't treat it specially.